manifests/user.pp

```diff
@@ -36,9 +39,10 @@
   Enum['present', 'absent', 'role'] $ensure              = 'present',
   String                            $username            = $title,
   String                            $group               = $title,
-  String                            $home                = "/home/${title}",
+  Stdlib::AbsolutePath              $home                = "/home/${title}",
```

Comment:
The reason I didn't use this is that there is no dependency on stdlib. If you do this, you must add it in `metadata.json`.